examples/others/fragment/lv_example_fragment_2.c

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
It crashes if I press "Push" and "Pop" after that. (I.e. when the last fragment is popped)

Reply:
Sorry for this issue, it's fixed.